components/ww3/cime_config/config_compsets.xml

```diff
@@ -30,8 +30,8 @@
   </compset>
 
   <compset>
-    <alias>GMPAS-JRA1p4-WW3</alias>
-    <lname>2000_DATM%JRA-1p4-2018_SLND_MPASSI_MPASO%DATMFORCED_DROF%JRA-1p4-2018_SGLC_WW3%sp36x36</lname>
+    <alias>GMPAS-JRA-WW3</alias>
+    <lname>2000_DATM%JRA-2018_SLND_MPASSI_MPASO%DATMFORCED_DROF%JRA-2018_SGLC_WW3%sp36x36</lname>
```

Comment:
@sbrus89 I think you want `JRA-2018` -> `JRA-1p5` in the two instances here for JRA v1.5. If you did want JRA v1.4 instead, than the previous `JRA-1p4-2018` is the correct one.